mlog/log.go

```diff
@@ -24,6 +27,19 @@ const (
 	LevelWarn = "warn"
 	// Errors are messages about things we know are problems
 	LevelError = "error"
+
+	// DefFlushTimeout is the default amount of time mlog.Flush will wait
+	// before timing out.
+	DefFlushTimeout = time.Second * 5
```

Comment:
Nit: `DefaultFlushTimeout` is more readable.

Reply:
Fixed:  https://github.com/mattermost/mattermost-server/pull/15398/commits/62c5a4cde5dd7dc36a4e32128ac5fbedec21962f